README.md

```diff
@@ -1,121 +1,70 @@
 # CDF Fabric replicator
 
-Application which utilizes the CDF APIs to replicate data to and from Microsoft Fabric
+Application that utilizes the CDF APIs to replicate data to and from Microsoft Fabric.
 
-# Replicator Services
+## Replicator Services
 The replicator consists of four services:
-- **Time series replicator** - Copies time series data from CDF to Fabric
-- **Data model replicator** - Copies data model nodes and edges from CDF to Fabric
-- **Event replicator** - Copies event data from CDF to Fabric
-- **Fabric data extractor** - Copies time series, events, and files from Fabric to CDF
+- **Time series replicator** - Copies time series data from CDF to Fabric.
+- **Data model replicator** - Copies data model nodes and edges from CDF to Fabric.
+- **Event replicator** - Copies event data from CDF to Fabric.
+- **Fabric data extractor** - Copies time series, events, and files from Fabric to CDF.
```

Comment:
A small nit, but should we use the full product names in this document, such as Microsoft Fabric, and Cognite Data Fusion

Reply:
Spot on. Thanks @wbevans